api/src/test/java/uk/gov/hmcts/payment/api/componenttests/CreditAccountPaymentControllerTest.java

```diff
@@ -1351,6 +1407,71 @@ private String creditAccountPaymentRequestJsonWithIAC_Json() {
             "}";
     }
 
+    private String creditAccountPaymentRequestJsonWithUnSpec_Json() {
+        return "{\n" +
+            "  \"amount\": 101.89,\n" +
+            "  \"description\": \"New passport application\",\n" +
+            "  \"ccd_case_number\": \"CCD101\",\n" +
+            "  \"case_reference\": \"12345\",\n" +
+            "  \"service\": \"UNSPEC\",\n" +
+            "  \"currency\": \"GBP\",\n" +
+            "  \"site_id\": \"AAA7\",\n" +
+            "  \"customer_reference\": \"CUST101\",\n" +
+            "  \"organisation_name\": \"ORG101\",\n" +
+            "  \"account_number\": \"AC101010\",\n" +
+            "  \"fees\": [\n" +
+            "    {\n" +
+            "      \"calculated_amount\": 101.89,\n" +
+            "      \"code\": \"X0101\",\n" +
+            "      \"version\": \"1\"\n" +
+            "    }\n" +
+            "  ]\n" +
+            "}";
+    }
+
+    private String creditAccountPaymentRequestJsonWithDifferentSiteIdForUnSpec_Json() {
```

Comment:
Instead of using hardcoded jsons for testing all the negative scenarios, can we 
1. convert a sample json into a Pojo
2. replace the data point required for the test scenario
3. convert the Pojo back to Json & pass through API

Reply:
Points noted. We will look into this in the next release.